web-components/src/components/tabs/tokens/md-tabs-tokens.js

```diff
@@ -27,12 +27,8 @@ const tabs = {
       dark: colors.gray[90].name
     },
     "focus-border": {
-      light: "transparent",
-      dark: "transparent"
-    },
-    "focus-shadow": {
-      light: "0 0 4px 2px $md-blue-50",
-      dark: "0 0 4px 2px $md-blue-50"
```

Comment:
@boiarchukk Can you provide the reasoning here? I winder if we're breaking a convention that was arrived at earlier .

Reply:
@futurethang In Momentum design we didn't have focus style for Tab, rather then in Lumos